<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rasterCalcDialog</class>
 <widget class="QDialog" name="rasterCalcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raster Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="BodyLabel" name="BodyLabel">
           <property name="text">
            <string>Raster Bands</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ListView" name="ListView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="CardWidget" name="CardWidget">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="BodyLabel" name="BodyLabel_4">
            <property name="text">
             <string>Result Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="resLE">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ToolButton" name="selectSaveFilePB"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget_2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="BodyLabel" name="BodyLabel_2">
            <property name="text">
             <string>Target Crs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="targetCrsLE"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CardWidget" name="CardWidget_5">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="PrimaryPushButton" name="selectMapcanvasCrsPB">
            <property name="text">
             <string>Use Current Crs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimarySplitPushButton" name="selectLayerSPB">
            <property name="text_" stdset="0">
             <string>Select Layer Crs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="selectMorePB">
            <property name="text">
             <string>Select...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="2" column="5">
      <widget class="PushButton" name="lnPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PushButton" name="minusPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="PushButton" name="sqrtPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="PushButton" name="paiPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ï€</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="PushButton" name="ePb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="PushButton" name="leftPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="PushButton" name="log10Pb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>log10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="PushButton" name="plusPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="PushButton" name="absPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>abs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="PushButton" name="atanPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="PushButton" name="cosPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="PushButton" name="rightPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="PushButton" name="notEqPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>!=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PushButton" name="mulPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="PushButton" name="orPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="PushButton" name="minPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MIN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="PushButton" name="sinPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="PushButton" name="acosPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="PushButton" name="maxPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MAX</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="PushButton" name="eqPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="PushButton" name="powPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>pow</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="PushButton" name="andPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AND</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="PushButton" name="tanPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="PushButton" name="greaterPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="PushButton" name="ifPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>if</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="PushButton" name="lessPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="PushButton" name="divPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="PushButton" name="greaterEqPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&gt;=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="PushButton" name="lessEqPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;=</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="PushButton" name="asinPb">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="4">
      <widget class="PrimaryPushButton" name="selectCalcTemplatePb">
       <property name="text">
        <string>Quickly Enter Exponential Calculation Formula</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TextEdit" name="TextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BodyLabel" name="expressionLabel">
     <property name="text">
      <string>Expression is Invalid!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="runPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SplitPushButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimarySplitPushButton</class>
   <extends>SplitPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
